### PR TITLE
slight copy adjustments on the index

### PR DESCRIPTION
**Pull request by bradnoble**
- Removed italics around mentions of pgBackRest (we don't italicize product names anywhere else)
- switched from "Postgres" to "PostgreSQL"
- put the link to the .org in a Markdown admonition (!!! Note...), per the new standard
- removed language in the intro about pgBackRest being "as easy as could be" as this is for users and not marketing per se

### CODE REVIEW

**Review by epbarger — APPROVED** [review]

**Comment by pgstef** [comment]
@bradnoble
Please don't merge any change before @sebasmannem or me would approve it please.

_Postgres_ was used on purpose. Meaning PostgreSQL + EPAS.
_pgBackRest_ was also intended on purpose. If you change it there, you'll have to change it in all the files. Now we're in an inconsistency state.

So please, revert this or correct it.

**Comment by epbarger** [comment]
My mistake, I jumped the gun on merging this

**Comment by pgstef** [comment]
> My mistake, I jumped the gun on merging this

Don't worry, PR #984 should fix it ;-)

**Pull request merged**
